src/boards/include/boards/adafruit_fruit_jam.h

```diff
@@ -28,6 +28,67 @@
 // This means RP2350B.
 #define PICO_RP2350A 0
 
+// --- BOARD SPECIFIC ---
+// On JST PH connector.
+#define ADAFRUIT_FRUIT_JAM_A0_PIN 40
+
+#define ADAFRUIT_FRUIT_JAM_A1_PIN 41
+#define ADAFRUIT_FRUIT_JAM_A2_PIN 42
+#define ADAFRUIT_FRUIT_JAM_A3_PIN 43
+#define ADAFRUIT_FRUIT_JAM_A4_PIN 44
+#define ADAFRUIT_FRUIT_JAM_A5_PIN 45
+
+#define ADAFRUIT_FRUIT_JAM_D6_PIN 6
+#define ADAFRUIT_FRUIT_JAM_D7_PIN 7
+#define ADAFRUIT_FRUIT_JAM_D8_PIN 8
+#define ADAFRUIT_FRUIT_JAM_D9_PIN 9
+#define ADAFRUIT_FRUIT_JAM_D10_PIN 10
+
+#define ADAFRUIT_FRUIT_JAM_BOOT_BUTTON_PIN 0
+
+#define ADAFRUIT_FRUIT_JAM_BUTTON1_PIN 4
+#define ADAFRUIT_FRUIT_JAM_BUTTON2_PIN 5
```

Comment:
No BUTTON3 ?

Reply:
Ehh, the boot button is one of the three buttons, but the internal nomenclature is a bit of a moving target, sorry! There are three buttons and their numbering is different. You'll see a fix.

Reply:
> the internal nomenclature is a bit of a moving target

In that case should we mark this PR as "Draft", until things are a bit more finalised?